include/benchmark/benchmark.h

```diff
@@ -167,21 +167,23 @@ BENCHMARK(BM_test)->Unit(benchmark::kMillisecond);
 #define BENCHMARK_HAS_CXX11
 #endif
 
-#include <stdint.h>
-
-#include <algorithm>
-#include <cassert>
-#include <cstddef>
-#include <iosfwd>
-#include <map>
-#include <set>
-#include <string>
-#include <vector>
+#include <stdint.h>  // IWYU pragma: keep
+
+#include <algorithm>  // for max
```

Comment:
does IWYU add the "// for ..." comments?

Reply:
By default, yes. I can drop those `// for ..` if wanted.

Reply:
please do. it's adding noise to the diff and doesn't actually add anything. If we trust IWYU then we trust it; we don't need to document its decision-making.

Reply:
i'm still seeing the comments.